# HG changeset patch
# User Rik <rik@octave.org>
# Date 1473385555 25200
#      Thu Sep 08 18:45:55 2016 -0700
# Node ID ca01dc025e2c80a8f470be79b143f9ac281bd5e0
# Parent  3a8af9d517de488e3d086c77e1d0556f61960e9f
Rename mxUNUSED_CLASS to mxVOID_CLASS for Matlab compatibility.

* mxarray.in.h: Rename mxUNUSED_CLASS to mxVOID_CLASS for Matlab compatibility.

diff --git a/libinterp/corefcn/mxarray.in.h b/libinterp/corefcn/mxarray.in.h
--- a/libinterp/corefcn/mxarray.in.h
+++ b/libinterp/corefcn/mxarray.in.h
@@ -58,17 +58,17 @@ mxComplexity;
 
 typedef enum
 {
   mxUNKNOWN_CLASS = 0,
   mxCELL_CLASS,
   mxSTRUCT_CLASS,
   mxLOGICAL_CLASS,
   mxCHAR_CLASS,
-  mxUNUSED_CLASS,
+  mxVOID_CLASS,
   mxDOUBLE_CLASS,
   mxSINGLE_CLASS,
   mxINT8_CLASS,
   mxUINT8_CLASS,
   mxINT16_CLASS,
   mxUINT16_CLASS,
   mxINT32_CLASS,
   mxUINT32_CLASS,
